src/worker/tasks/sendWebhookWorker.ts

```diff
@@ -11,36 +16,53 @@ import {
   WebhookJob,
 } from "../queues/sendWebhookQueue";
 
-interface WebhookBody {
-  type: "event-log" | "transaction-receipt";
-  data: any;
-}
-
 const handler: Processor<any, void, string> = async (job: Job<string>) => {
   const { data, webhook } = superjson.parse<WebhookJob>(job.data);
 
   let resp: WebhookResponse | undefined;
-  if (data.type === WebhooksEventTypes.CONTRACT_SUBSCRIPTION) {
-    let webhookBody: WebhookBody;
-    if (data.eventLog) {
-      webhookBody = {
-        type: "event-log",
-        data: toEventLogSchema(data.eventLog),
-      };
-    } else if (data.transactionReceipt) {
-      webhookBody = {
-        type: "transaction-receipt",
-        data: toTransactionReceiptSchema(data.transactionReceipt),
+  switch (data.type) {
+    case WebhooksEventTypes.CONTRACT_SUBSCRIPTION: {
+      let webhookBody: {
+        type: "event-log" | "transaction-receipt";
+        data: any;
       };
-    } else {
-      throw new Error(
-        'Missing "eventLog" or "transactionReceipt" for CONTRACT_SUBSCRIPTION webhook.',
-      );
+      if (data.eventLog) {
+        webhookBody = {
+          type: "event-log",
+          data: toEventLogSchema(data.eventLog),
+        };
+      } else if (data.transactionReceipt) {
+        webhookBody = {
+          type: "transaction-receipt",
+          data: toTransactionReceiptSchema(data.transactionReceipt),
+        };
+      } else {
+        throw new Error(
+          'Missing "eventLog" or "transactionReceipt" for CONTRACT_SUBSCRIPTION webhook.',
+        );
+      }
+      resp = await sendWebhookRequest(webhook, webhookBody);
+      break;
     }
-    resp = await sendWebhookRequest(webhook, webhookBody);
+
+    case WebhooksEventTypes.ALL_TX:
+    case WebhooksEventTypes.QUEUED_TX:
+    case WebhooksEventTypes.SENT_TX:
+    case WebhooksEventTypes.MINED_TX:
+    case WebhooksEventTypes.ERRORED_TX:
+    case WebhooksEventTypes.CANCELLED_TX: {
+      const webhookBody: Static<typeof transactionResponseSchema> =
+        toTransactionSchema(data.transaction);
+      resp = await sendWebhookRequest(webhook, webhookBody);
+      break;
+    }
+
+    default:
+      console.log(`Unknown webhook type: ${(data as any).type}`);
+      return;
   }
 
-  if (resp && !resp.ok) {
+  if (!resp.ok) {
```

Comment:
resp has type `WebhookResponse | undefined`. would this change cause unexpected behavior if resp is undefined?